app/src/androidTest/java/com/example/quizacademytask/MigrationTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.quizacademytask
+
+import androidx.room.migration.Migration
+import androidx.room.testing.MigrationTestHelper
+import androidx.sqlite.db.SupportSQLiteDatabase
+import androidx.sqlite.db.framework.FrameworkSQLiteOpenHelperFactory
+import androidx.test.platform.app.InstrumentationRegistry
+import db.AppDatabase
+import junit.framework.Assert.*
+import org.junit.Rule
+import org.junit.Test
+
+class MigrationTest {
+    private val testDBName = "test_db"
+
+    @JvmField
+    @Rule
+    val testHelper: MigrationTestHelper = MigrationTestHelper(
+        InstrumentationRegistry.getInstrumentation(),
+        AppDatabase::
+        class.java.canonicalName, FrameworkSQLiteOpenHelperFactory()
+    )
+
+    private val MIGRATION_1_2: Migration = object : Migration(1, 2) {
```

Comment:
Migration should be part of AppDatabase and not defined in test. Test should use real migrations.